bin/test_piotr.py

```diff
@@ -0,0 +1,8 @@
+import requests
+f = open('../test/queries/english_00.txt', 'r')
```

Comment:
Can you use this to read any of the files present in the `queries` directory, read each file and fire queries for each query in the file. That way its a generic script to fire anything in the `queries` directory.